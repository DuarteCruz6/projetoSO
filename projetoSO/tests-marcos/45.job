WRITE [(m9,40)(a7,4)]
READ [a7]
DELETE [m9]
READ [a7]
DELETE [a7]
BACKUP
SHOW
SHOW
WRITE [(x5,38)]
WRITE [(d2,84)(u8,22)]
READ [u8]
READ [x5]
WRITE [(b9,87)(n1,39)(o0,47)(s3,99)]
SHOW
WRITE [(o3,13)(c5,26)(p9,35)(u5,65)(s9,65)]
WRITE [(v9,56)(e3,84)]
WRITE [(l1,88)(t3,42)(a2,30)(v1,84)(z0,84)]
DELETE [l1,p9,x5]
READ [n1]
WRITE [(n8,87)(o4,85)(c0,72)(r1,49)]
READ [n1]
WRITE [(o7,12)(x8,40)(w4,99)(k4,55)]
WRITE [(y9,4)(c4,92)(a7,63)(i1,12)(t6,75)]
WAIT 3000
READ [v1]
DELETE [y9]
READ [o4]
WRITE [(g8,99)]
BACKUP
WAIT 1000
WRITE [(r9,6)(f6,7)(h6,96)(y3,10)(p5,6)]
WRITE [(b5,3)(b6,41)(n9,79)(a6,3)(q8,9)]
WRITE [(d4,8)(u7,39)(x5,41)]
READ [t6]
DELETE [c5,k4]
WRITE [(d5,38)]
READ [c4]
SHOW
WRITE [(p4,36)(q7,24)(o9,99)]
DELETE [c4,d4]
WRITE [(k6,6)(m5,27)(c7,23)(e6,59)]
WRITE [(n3,44)(u0,53)(w1,31)]
WRITE [(y9,52)(n6,26)(p3,81)]
READ [w4]
WRITE [(f4,64)(d0,73)(b4,67)]
DELETE [o4]
READ [u5]
WRITE [(m1,1)(w9,71)(z8,98)(l5,68)(u1,65)]
READ [e3]
WRITE [(u3,61)(g3,59)(k1,94)(h1,10)(g7,84)]
