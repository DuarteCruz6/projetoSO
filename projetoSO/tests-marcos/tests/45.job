WRITE [(d5,53)]
WRITE [(p3,88)(j4,26)(w9,26)(o0,84)(o0,88)]
WRITE [(y4,76)(e2,57)]
SHOW
WAIT 2000
READ [o0]
WRITE [(n5,45)(p9,100)(o4,52)(o2,33)]
DELETE [y4,e2,p3]
WRITE [(f0,62)(x4,5)(g0,10)(h0,20)(d9,92)]
READ [p9]
BACKUP
READ [p9]
WRITE [(w5,92)(e5,96)(x7,19)(q2,80)(g5,55)]
WRITE [(f3,4)]
DELETE [w9]
WRITE [(v0,32)(i8,55)(t1,62)]
DELETE [x4,f0,w5]
READ [g5]
DELETE [i8,x7]
DELETE [g5]
WRITE [(i5,32)(z8,25)(q9,61)(q8,88)]
DELETE [j4]
SHOW
READ [q9]
READ [o0]
WRITE [(g4,49)(x3,55)(w1,43)]
READ [f3]
WRITE [(b9,60)(e0,14)(m5,47)(l3,7)(c4,28)]
WRITE [(y9,62)]
WRITE [(n4,23)]
READ [n4]
WRITE [(s4,55)(i4,74)(t2,58)]
WRITE [(a3,29)(y1,23)]
WAIT 1000
WRITE [(j5,49)(m7,30)(l2,91)]
BACKUP
WRITE [(l7,84)]
WRITE [(e7,14)(t6,68)(v6,3)(j3,52)(b0,84)]
WRITE [(u9,38)]
WRITE [(f5,75)(u6,52)(w2,60)(w0,54)(f7,16)]
READ [b9]
SHOW
WRITE [(y8,50)]
WRITE [(n7,94)(h7,60)(c6,15)]
READ [i4]
DELETE [b9,d9]
READ [v6]
READ [w1]
WRITE [(w6,87)(n0,25)(q1,19)]
DELETE [x3]
READ [w1]
WAIT 1000
SHOW
READ [n5]
DELETE [q2,y1,u6,e0,c4]
WRITE [(s5,99)(q2,29)]
WRITE [(c9,4)(b1,93)(s8,0)(f0,23)]
BACKUP
WRITE [(o6,14)(e1,96)(u7,23)]
WRITE [(p5,12)(c3,59)(n6,69)(r0,33)]
READ [l3]
WRITE [(v7,66)(j4,98)]
SHOW
WRITE [(y7,3)]
WRITE [(p3,99)(g7,47)(w7,60)]
WRITE [(t8,67)(b6,37)]
DELETE [q2,o0,t2,e7]
DELETE [g4,n0,g0,z8,y8]
WRITE [(k0,50)(i1,86)]
WRITE [(w4,23)(d1,91)]
READ [h7]
WAIT 2000
WAIT 3000
DELETE [o4]
WRITE [(v9,82)(r6,75)]
READ [c6]
DELETE [h0,j5]
WRITE [(r4,59)(d4,21)(o3,37)(r7,58)]
READ [m5]
READ [q9]
WRITE [(v4,28)(z8,17)(e8,47)(f8,68)]
READ [f5]
DELETE [r0,h7]
WRITE [(g1,35)(u6,32)(t9,66)]
WRITE [(p2,68)(x9,11)(r1,40)(a6,45)(o1,72)]
WRITE [(k6,48)(x4,100)(h2,5)(g9,64)(k8,28)]
READ [r4]
WRITE [(z7,8)(w9,13)(p0,96)(q4,81)(n2,50)]
WRITE [(n0,59)(s3,54)(e7,81)]
