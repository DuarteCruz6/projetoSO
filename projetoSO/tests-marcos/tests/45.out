(d5, 53)
(e2, 57)
(j4, 26)
(o0, 88)
(p3, 88)
(w9, 26)
(y4, 76)
[(o0,88)]
[(p9,100)]
[(p9,100)]
[(g5,55)]
(d9, 92)
(d5, 53)
(e5, 96)
(f3, 4)
(g0, 10)
(h0, 20)
(i5, 32)
(n5, 45)
(o2, 33)
(o4, 52)
(o0, 88)
(p9, 100)
(q8, 88)
(q9, 61)
(q2, 80)
(t1, 62)
(v0, 32)
(z8, 25)
[(q9,61)]
[(o0,88)]
[(f3,4)]
[(n4,23)]
[(b9,60)]
(a3, 29)
(b0, 84)
(b9, 60)
(c4, 28)
(d9, 92)
(d5, 53)
(e7, 14)
(e0, 14)
(e5, 96)
(f7, 16)
(f5, 75)
(f3, 4)
(g4, 49)
(g0, 10)
(h0, 20)
(i4, 74)
(i5, 32)
(j3, 52)
(j5, 49)
(l7, 84)
(l2, 91)
(l3, 7)
(m7, 30)
(m5, 47)
(n4, 23)
(n5, 45)
(o2, 33)
(o4, 52)
(o0, 88)
(p9, 100)
(q8, 88)
(q9, 61)
(q2, 80)
(s4, 55)
(t6, 68)
(t2, 58)
(t1, 62)
(u6, 52)
(u9, 38)
(v6, 3)
(v0, 32)
(w0, 54)
(w2, 60)
(w1, 43)
(x3, 55)
(y1, 23)
(y9, 62)
(z8, 25)
[(i4,74)]
[(v6,3)]
[(w1,43)]
[(w1,43)]
(a3, 29)
(b0, 84)
(c6, 15)
(c4, 28)
(d5, 53)
(e7, 14)
(e0, 14)
(e5, 96)
(f7, 16)
(f5, 75)
(f3, 4)
(g4, 49)
(g0, 10)
(h7, 60)
(h0, 20)
(i4, 74)
(i5, 32)
(j3, 52)
(j5, 49)
(l7, 84)
(l2, 91)
(l3, 7)
(m7, 30)
(m5, 47)
(n0, 25)
(n7, 94)
(n4, 23)
(n5, 45)
(o2, 33)
(o4, 52)
(o0, 88)
(p9, 100)
(q1, 19)
(q8, 88)
(q9, 61)
(q2, 80)
(s4, 55)
(t6, 68)
(t2, 58)
(t1, 62)
(u6, 52)
(u9, 38)
(v6, 3)
(v0, 32)
(w6, 87)
(w0, 54)
(w2, 60)
(w1, 43)
(y8, 50)
(y1, 23)
(y9, 62)
(z8, 25)
[(n5,45)]
[(l3,7)]
(a3, 29)
(b1, 93)
(b0, 84)
(c3, 59)
(c9, 4)
(c6, 15)
(d5, 53)
(e1, 96)
(e7, 14)
(e5, 96)
(f0, 23)
(f7, 16)
(f5, 75)
(f3, 4)
(g4, 49)
(g0, 10)
(h7, 60)
(h0, 20)
(i4, 74)
(i5, 32)
(j4, 98)
(j3, 52)
(j5, 49)
(l7, 84)
(l2, 91)
(l3, 7)
(m7, 30)
(m5, 47)
(n6, 69)
(n0, 25)
(n7, 94)
(n4, 23)
(n5, 45)
(o6, 14)
(o2, 33)
(o4, 52)
(o0, 88)
(p5, 12)
(p9, 100)
(q2, 29)
(q1, 19)
(q8, 88)
(q9, 61)
(r0, 33)
(s8, 0)
(s5, 99)
(s4, 55)
(t6, 68)
(t2, 58)
(t1, 62)
(u7, 23)
(u9, 38)
(v7, 66)
(v6, 3)
(v0, 32)
(w6, 87)
(w0, 54)
(w2, 60)
(w1, 43)
(y8, 50)
(y9, 62)
(z8, 25)
[(h7,60)]
[(c6,15)]
[(m5,47)]
[(q9,61)]
[(f5,75)]
[(r4,59)]
